RELEASE_PROCESS.md

```diff
@@ -39,15 +41,34 @@ Every commit to chef-server master is tested against a full pedant
 run. However, upgrade testing must still be done in advance of the
 release:
 
-- [ ] Run the Chef Delivery-based automated upgrade testing
-  * Checkout the `chef-server-acceptance` project from delivery.chef.co
-  * Update the `chef_server_test_url-override` attributes in
-    `.delivery/build/attributes/default.rb` to the URL corresponding to
-    the latest package you'd like to test at artifactory.chef.co
-  * Commit the change and run `delivery review`
+- [ ] Run the Chef Delivery-based automated upgrade testing.
+  * Checkout the `chef-server-acceptance` project from delivery.chef.co.
+  * Update relevant attributes in `.delivery/build/attributes/default.rb`:
+    * Update `chef_server_test_url-override` to be the URL corresponding to
+      the latest package you'd like to test at artifactory.chef.co (probably
+      a git poll build that you want to ship as the next version).
+    * Update `chef_server_latest_released_version` to match the last release of
+      the Chef Server. Should match the version at the [download page](https://downloads.chef.io/chef-server/).
+    * If delivery.chef.co builders are too constrainted to run the full matrix of 10 jobs
+      (ask in #chef-server or #delivery-support if you are unsure what this means),
+      comment the second half of the entries in the
+      `default['delivery-matrix']['acceptance']['matrix']` array and run a build. After that
+      build and all its matrix builds finish, run another build with the first half
+      commented and the second half uncommented (see below for where to find matrix builds).
+      Regardless, you should make sure every matrix entry gets run before you ship,
+      so always double-check these attributes, even if running the full matrix, as some
+      might be commented out from previous runs.
+  * Commit the change and run `delivery review`.
   * When the lint and unit tests pass, approve the change and watch the
-    matrix of tests run
-
+    matrix of tests run. This change will spin off several new changes, so look for
+    them in the root of the project in the delivery UI.
+    After your change, you should see a change called "Merged change..." per entry of the matrix
+    you have uncommented.
+  * If you had no failures, AWS instances will be cleaned up for you and the cookbook cleans up any
+    instances over 24 hours old the next time it runs, but it's good to log into AWS and look for any machine instances
+    tagged with `qa-chef-server-cluster-delivery-builder` in `US West (Oregon) us-west-2` region
```

Comment:
I presume these are defined in the build cookbook somewhere/somewhow?  I couldn't find it with a naive git search.